Sources/StreamChatUI/ChatChannelList/ChatChannelListSkeletonView.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ChatChannelListSkeletonView.swift
+//  StreamChat
+//
+//  Created by Hugo Bernal on 21/07/22.
+//  Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+open class ChatChannelListSkeletonView: _View, ThemeProvider, UITableViewDataSource {
```

Comment:
Same here, documentation here would be nice 👍

Reply:
Added